kernel/tests/s04_user_runtime_tests.rs

```diff
@@ -31,6 +31,7 @@ fn s04_userspace_multicore() {
         .user_feature("test-scheduler-smp")
         .build();
     let cmdline = RunnerArgs::new_with_build("userspace-smp", &build)
+        .nodes(num_cores / 16)
```

Comment:
Does this no longer work with full cores? It's unclear to me how 16 plays into this test.